Kiosk/App/Models/Artwork.swift

```diff
@@ -30,13 +46,14 @@ public class Artwork: JSONAble {
         return defaultImages?.first ?? self.images?.first
     }()
 
-    init(id: String, dateString: String, title: String, titleAndDate: NSAttributedString, price: String, date: String) {
+    init(id: String, dateString: String, title: String, titleAndDate: NSAttributedString, price: String, date: String, sold: String) {
```

Comment:
That's a good question. Classes do allow you to do this, but only for `var` properties that have default values (it'll generate an `init()` function). So this wouldn't work:

```swift
class Thing {
    let t: String
}

let thing = Thing(t: "hi") // 😓
```

However, _structs_ will do this fine. 

```swift
struct Thing {
    let t: String
}

let thing = Thing(t: "hi")
```